wasm/public/wasm_shell.html

```diff
@@ -1,102 +1,158 @@
+        <a href="help@schrodinger.com">help@schrodinger.com</a> and provide the following error
+        message:
+      </p>
+      <div style="background: #f5f5f5; padding: 16px; border-radius: 5px">
+        <div id="error-text" style="max-height: 200px; overflow: hidden"></div>
         <button
           onclick="copyErrorToClipboard()"
-          style="background:#fbfbfb; color:#505050; width:100%; padding:10px; margin-top:16px; border:1px solid #ebebeb; border-radius:5px; cursor:pointer; display: flex; align-items:center; justify-content:center; gap:3px;"
+          style="
+            background: #fbfbfb;
+            color: #505050;
+            width: 100%;
+            padding: 10px;
+            margin-top: 16px;
+            border: 1px solid #ebebeb;
+            border-radius: 5px;
+            cursor: pointer;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+            gap: 3px;
+          "
         >
           <img src="copy_icon.svg" /> Copy to Clipboard
         </button>
       </div>
-      <img src="schrodinger_logo.png" width="150" style="margin-top:auto; margin-bottom:10px;" />
+      <img src="schrodinger_logo.png" width="150" style="margin-top: auto; margin-bottom: 10px" />
     </div>
     <div id="screen"></div>
 
-    <script type='text/javascript'>
-        let qtLoader = undefined;
-        function init() {
-          var spinner = document.querySelector('#qtspinner');
-          var canvas = document.querySelector('#screen');
-          var status = document.querySelector('#qtstatus');
-          var error = document.querySelector('#qterror');
-          var errorTextContainer = document.querySelector('#error-text');
+    <script type="text/javascript" src="qtloader.js"></script>
+    <script type="text/javascript" src="sketcher_app.js"></script>
+    <script type="text/javascript">
+      async function init() {
+        const spinner = document.querySelector('#qtspinner');
+        const status = document.querySelector('#qtstatus');
+        const error = document.querySelector('#qterror');
+        const errorTextContainer = document.querySelector('#error-text');
+        const container = document.querySelector('#screen');
 
-          // SKETCH-2309: lock down the value to prevent distortion when zooming
-          window.devicePixelRatio = Math.round(window.devicePixelRatio * 2) / 2;
+        // SKETCH-2309: lock down the value to prevent distortion when zooming
+        window.devicePixelRatio = Math.round(window.devicePixelRatio * 2) / 2;
 
-          qtLoader = new QtLoader({
-              canvasElements : [canvas],
-              showLoader: function(loaderStatus) {
-                  spinner.style.display = 'flex';
-                  canvas.style.display = 'none';
-                  error.style.display = 'none';
-                  status.innerHTML = loaderStatus + "...";
```

Comment:
is displaying the loaderStatus something we can/should still do? The text "Downloading..." and "Compiling..." are gone without it, but not sure how much that really matters?